plugin-api/src/main/java/org/hyperledger/besu/plugin/services/securitymodule/PublicKey.java

```diff
@@ -12,7 +12,7 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package org.hyperledger.besu.crypto;
+package org.hyperledger.besu.plugin.services.securitymodule;
 
 import org.apache.tuweni.bytes.Bytes;
 
```

Comment:
This should be an interface not a class.

Reply:
refactored as interface.